src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/custom.py

```diff
@@ -75,30 +78,42 @@ def update_account(client, resource_group_name, account_name,
                          auto_storage=properties)
 
 
-def login_account(client, resource_group_name, account_name, shared_key_auth=False):
-
+def login_account(cmd, client, resource_group_name, account_name, shared_key_auth=False):
```

Comment:
@xingwu1  and @alfpark - these are the changes to the login command to return the token info for you to review. Are you happy with me also returning the account keys if --shared-key-auth is specified?

Reply:
@annatisch do we want to have a flag to indicate whether it will have response? The response contains secret, only explicit flag will dump the secret (key/token).

Everything else looks good to me.